src/applicants/applicant.entity.ts

```diff
@@ -0,0 +1,25 @@
+import {Entity, Column, PrimaryGeneratedColumn} from "typeorm"
+
+@Entity()
+export class Applicant {
+    @PrimaryGeneratedColumn('uuid')
+    id: string
+
+    @Column({
+        unique: true
+    })
+    email: string;
+
+
+    @Column({
+        type: 'enum',
+        enum: ['male', 'female']
+    })
+    gender: string;
+
+    @Column({
+        type: 'enum',
+        enum: ['web development', 'Machine Learning', 'Data Science']
```

Comment:
There should be an entity in the database to store these values. probably named Internship Categories and then a relationship between this and column and that table